arches_lingo/src/arches_lingo/types.ts

```diff
@@ -98,6 +98,18 @@ export interface AppellativeStatus extends TileData {
     appellative_status_timespan_end_of_the_end: string;
 }
 
+export interface ConceptStatement extends TileData {
+    statement_content: string;
+    statement_language?: ControlledListItem[];
+    statement_type?: ControlledListItem[];
+    statement_type_metatype?: ControlledListItem[];
+    statement_data_assignment_object_used: ResourceInstanceReference[];
+    statement_data_assignment_actor: ResourceInstanceReference[];
+    statement_data_assignment_type: ControlledListItem[];
+    statement_data_assignment_timespan_begin_of_the_begin: string;
+    statement_data_assignment_timespan_end_of_the_end: string;
```

Comment:
Everything except statement_content is not required, so I'd make these optional
```suggestion
    statement_data_assignment_object_used?: ResourceInstanceReference[];
    statement_data_assignment_actor?: ResourceInstanceReference[];
    statement_data_assignment_type?: ControlledListItem[];
    statement_data_assignment_timespan_begin_of_the_begin?: string | null;
    statement_data_assignment_timespan_end_of_the_end?: string | null;
```